apps/graphql/src/apps/itinerary/types/inputs/ItineraryCheckInput.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import {
+  GraphQLInputObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLInt,
+} from 'graphql';
+
+import PassengersInput from './PassengersInput';
+
+export default new GraphQLInputObjectType({
+  name: 'ItineraryCheckInput',
+  fields: {
+    bookingToken: {
```

Comment:
I can find any bookingToken field in BookingInterface Schema, so I a not sure if this is the right name. Where I can find bookingToken in the graphQL Schema?

Reply:
It's not on Booking but on Itinerary. The name makes sense to me, because it serves for itinerary booking. But we can definitely change it if you have something better in mind.

Reply:
make me more sense now thanks